### PR TITLE
TSK-1349: Added unit test for classification-list component

### PR DESCRIPTION
**Pull request by sofie29**
sonar: https://sonarcloud.io/dashboard?branch=TSK-1349&id=sofie29_taskana

documentation: https://taskana.atlassian.net/browse/TSK-1349

### CODE REVIEW

File: web/src/app/administration/components/classification-list/classification-list.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+  it('should display import-export component', () => {
+    expect(debugElement.nativeElement.querySelector('taskana-administration-import-export')).toBeTruthy();
+  });
+
+  it('should display classification-types-selector component', () => {
+    const typesSelectorComponent = debugElement.nativeElement.querySelector(
+      'taskana-administration-classification-types-selector'
+    );
+    expect(typesSelectorComponent).toBeTruthy();
+  });
```
Comment (cnguyen-de) [inline web/src/app/administration/components/classification-list/classification-list.component.spec.ts:133]: hier das element in einem Variable gespeichert

File: web/src/app/administration/components/classification-list/classification-list.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+  /* HTML: FILTER */
+  it('should display specific icon when selectedCategory is true', () => {
+    component.selectedCategory = 'EXTERNAL';
+    fixture.detectChanges();
+    expect(debugElement.nativeElement.querySelector('.selected-category')).toBeTruthy();
```
Comment (cnguyen-de) [inline web/src/app/administration/components/classification-list/classification-list.component.spec.ts:139]: aber hier nicht?

File: web/src/app/administration/components/classification-list/classification-list.component.spec.ts (hunk 1 above)
Comment (sofie29) [reply web/src/app/administration/components/classification-list/classification-list.component.spec.ts:133]: This selector is so long that it wouldn't fit into one line. The other selector is shorter.

Comment (cnguyen-de) [reply web/src/app/administration/components/classification-list/classification-list.component.spec.ts:133]: ok

**Review by cnguyen-de — APPROVED** [review]
css class names could use BEM convention, but we're rewriting the html soon, it shouldn't matter atm.

**Pull request merged**